src/collectd.conf.in

```diff
@@ -103,7 +103,7 @@
 #@BUILD_PLUGIN_CONNTRACK_TRUE@LoadPlugin conntrack
 #@BUILD_PLUGIN_CONTEXTSWITCH_TRUE@LoadPlugin contextswitch
 @BUILD_PLUGIN_CPU_TRUE@@BUILD_PLUGIN_CPU_TRUE@LoadPlugin cpu
-#@BUILD_PLUGIN_CPUFREQ_TRUE@LoadPlugin cpufreq
+@BUILD_PLUGIN_CPUFREQ_TRUE@LoadPlugin cpufreq
```

Comment:
This enables the plugin by default. That's a change in behaviour and breaks on other platforms. Please remove it.